app/assets/javascripts/responsive-tables.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function() {
+  var switched = false;
+  var updateTables = function() {
+    if (($(window).width() < 767) && !switched ){
+      switched = true;
+      $('table.responsive').each(function(i, element) {
+        splitTable($(element));
+      });
+      return true;
+    }
+    else if (switched && ($(window).width() > 767)) {
+      switched = false;
+      $('table.responsive').each(function(i, element) {
+        unsplitTable($(element));
+      });
+    }
+  };
+
+  $(window).load(updateTables);
+  $(window).on('redraw',function(){switched=false;updateTables();});
+  $(window).on('resize', updateTables);
+
+
+	function splitTable(original)
+	{
+		original.wrap('<div class='table-wrapper' />');
+
+		var copy = original.clone();
+		copy.find('td:not(:first-child),
+      th:not(:first-child)').css('display', 'none');
+		copy.removeClass('responsive');
+
+		original.closest('.table-wrapper').append(copy);
+		copy.wrap('<div class='pinned' />');
```

Comment:
Expected ')' and instead saw 'pinned'.<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.<br>Expected an identifier and instead saw ')'.